Umbraco-Heartcore/Versions-and-updates/index.md

```diff
@@ -0,0 +1,17 @@
+---
+versionFrom: 8.0.0
+---
+
+# Versions and updates
+
+Umbraco Heartcore is a headless SaaS (software as a service) product, which means that one of the benefits is that you do not have to worry about upgrades. These will all be handled by Umbraco HQ. 
```

Comment:
Umbraco Heartcore is a software as a service (SaaS) offering, which means that one of hte benefits is that you do not need to worry about upgrades. All maintenance related to the cms, its features and APIs are handled by Umbraco HQ.